Questie.lua

```diff
@@ -128,5 +128,32 @@ function Questie:Debug(...)
     end
 end
 
+Questie.icons = {
```

Comment:
Not sure if this is the best place for this stuff, I mainly chose it because I didn't want to be loading modules all over the place. :shrug: